android/src/main/java/com/pw/droplet/braintree/Braintree.java

```diff
@@ -1,301 +1,316 @@
+
+                WritableMap billingContact = Arguments.createMap();
+
+                billingContact.putString(
+                        "streetAddress",
+                        ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getStreetAddress());
+                billingContact.putString(
+                        "city", ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getLocality());
+                billingContact.putString(
+                        "state", ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getRegion());
+                billingContact.putString(
+                        "country", ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getCountryCodeAlpha2());
+                billingContact.putString(
+                        "postalCode", ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getPostalCode());
+                billingContact.putString(
+                        "countryCode", ((GooglePayCardNonce) paymentMethodNonce)
+                                .getBillingAddress()
+                                .getCountryCodeAlpha2());
+                result.putMap("billingContact", billingContact);
+                mPromise.resolve(result);
+            }
+        }
     }
 
     @ReactMethod
-    public void showGooglePayViewController(final ReadableMap options,
-                                            final Callback successCallback,
-                                            final Callback errorCallback) {
-        this.successCallback = successCallback;
-        this.errorCallback = errorCallback;
-        GooglePaymentRequest googlePaymentRequest =
-                new GooglePaymentRequest()
-                        .transactionInfo(
-                                TransactionInfo.newBuilder()
-                                        .setTotalPrice(options.getString("totalPrice"))
-                                        .setTotalPriceStatus(
-                                                WalletConstants.TOTAL_PRICE_STATUS_FINAL)
-                                        .setCurrencyCode(options.getString("currencyCode"))
-                                        .build())
-                        .billingAddressRequired(options.getBoolean("requireAddress"));
-
-        if (options.hasKey("googleMerchantId")) {
-            googlePaymentRequest.googleMerchantId(
-                    options.getString("googleMerchantId"));
+    public void venmoRequest(final Promise promise) {
+        mPromise = promise;
+
+        if (mCurrentActivity != null) {
+            mVenmoClient = new VenmoClient(mBraintreeClient);
```

Comment:
Need to pass in context

    venmoClient = new VenmoClient(this, braintreeClient);